### PR TITLE
Follow orders

### PR DESCRIPTION
**Pull request by ihaywood3**
The "Follow Orders" bot. This is fundamentally the same as the liquidity "walls" in the original stakemachine, but battle-hardened after some (expensive) experiences running earlier versions on the AUD:BTS market.

### CODE REVIEW

**Comment by mikakoi** [comment]
Looks good, I'll take a more in-depth look at this once I find the time to merge the Easyconfig pull request

File: docs/follow_orders.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+**********************
+Follow Orders Strategy
+**********************
+
+This strategy places a buy and a sell walls into a specific market.
+It buys below a base price, and sells aboe the base price.
```
Comment (nikolaybaychenko) [inline docs/follow_orders.rst]: Typo at "aboe" should be "above"

File: dexbot/strategies/follow_orders.py [hunk 2]
```diff
@@ -5,79 +5,143 @@
...
-        
+
         myorders = {}
 
         if newprice < self.bot["min"]:
             self.disabled = True
-            self.log.critical("Price %f is below minimum %f" % (newprice,self.bot["min"]))
-            return
+            self.log.critical(
+                "Price %f is below minimum %f" %
```
Comment (mikakoi) [inline dexbot/strategies/follow_orders.py]: Please use .format() rather than %. Using % is an old relic from python 2

Comment (mikakoi) [reply dexbot/strategies/follow_orders.py]: (This applies to all the lines that use % for string formating)

File: dexbot/strategies/follow_orders.py [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                "float",
+                100.0,
+                "Starting price, as percentage of bid/ask spread",
+                (0.0,
+                 100.0)),
+            ConfigElement(
+                "reset",
+                "bool",
+                False,
+                "bot will alwys reset orders on start",
```
Comment (nikolaybaychenko) [inline dexbot/strategies/follow_orders.py]: Typo: should be "always" instead of "alwys"

**Comment by ihaywood3** [comment]
largely superseded by Relative orders/Staggered Orders

**Pull request closed without merging**